src/gc/gc.cpp

```diff
@@ -33623,6 +33620,8 @@ HRESULT GCHeap::Initialize ()
     }
 
     g_gc_pFreeObjectMethodTable = GCToEEInterface::GetFreeObjectMethodTable();
+    g_num_processors = GCToOSInterface::GetCurrentProcessCpuCount();
```

Comment:
Ok! In that case I'm not sure if there's an existing API on `GCToOSInterface` to do that. `GCToOSInterface::GetLogicalCpuCount` ends up calling [this](https://github.com/dotnet/coreclr/blob/master/src/vm/amd64/cgenamd64.cpp#L464). I remember us receiving some guidance from Intel about updating this function - do you think now would be a good time for that?